src/components/containers/recentArticles/RecentArticles.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import './recentArticles.css';
+import { recentArticlesAction } from '../../../actions/recentArticlesAction';
+import RecentArticleCard from './recentArticlesCard.jsx';
+
+class RecentArticles extends Component {
+  componentDidMount() {
+    this.props.recentArticlesAction();
+  }
+
+  render() {
+    return (
+      <section className="section mt-4 mobile-section">
+        <div className="container">
+          <div className="justify-space-between">
+            <div className="trending-articles">
+              <h1 className="title is-2 mb-3">
+                Recently Published Articles
+              </h1>
+            </div>
+          </div>
+          < div className = "columns is-variable is-8 mt-3">
+            {
+              this.props.articles
+            && this.props.articles.slice(0, 3).map((article, index) => {
```

Comment:
Good job using Slice here. However you shouldn't  do this in the component. Either at the reducer or the store. Also shouldn't the api be returning the exact number of articles you need?

Reply:
The get all articles api endpoint returns more than what I need. So I'm using the slice to display what I need

Reply:
The slice is just for display. The bulma style is expecting three card in one row.